content/recipes/prisma.md

```diff
@@ -1,98 +1,98 @@
 ### Prisma
 
-[Prisma](https://www.prisma.io/) turns your database into a GraphQL API and enables the use of GraphQL as a universal query language for all databases. Instead of writing SQL or using a NoSQL API, you can query your database with GraphQL. In this chapter we won't go into details about Prisma, so head over to their website and have a look what features are available.
+[Prisma](https://www.prisma.io/) provides access to your database via a powerful type-safe database client. Instead of writing SQL or using a NoSQL API, you can query your database with auto-generated crud operations exposed in the Prisma package (e.g. prisma.user.findOne({ where: { id: "someid" } })). In this chapter we won't go into details about Prisma, so head over to their website and have a look what features are available.
```

Comment:
```suggestion
[Prisma](https://www.prisma.io/) provides access to your database via a powerful type-safe database client. Instead of writing SQL or using a NoSQL API, you can query your database with auto-generated crud operations exposed in the Prisma package (e.g., `prisma.user.findOne({{ '{' }} where: {{ '{' }} id: "someid" {{ '}' }} {{ '}' }}))`. In this chapter we won't go into details about Prisma, so head over to [their website](https://www.prisma.io/) and have a look at what features are available.
```